rules/ember.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  plugins: [
+    'ember',
+  ],
+  rules: {
+    // General
+    'ember/local-modules': 'off',
+    'ember/no-observers': 'off',
+    'ember/no-side-effect': 'off',
+    'ember/jquery-ember-run': 'off',
```

Comment:
Should this be switched on? Not sure whether there are good use cases for omitting the run loop.